src/core/dom/image/const.ts

```diff
@@ -0,0 +1,53 @@
+/*!
+ * V4Fire Client Core
+ * https://github.com/V4Fire/Client
+ *
+ * Released under the MIT license
+ * https://github.com/V4Fire/Client/blob/master/LICENSE
+ */
+
+import { DefaultParams } from 'core/dom/image';
```

Comment:
Внутри модуля юзай точные указания пути, чтобы случайно не создать цикл.

```suggestion
import { DefaultParams } from 'core/dom/image/interface';
```